event.php

```diff
@@ -186,11 +186,11 @@
     <?php if ($access_level >= 2) : ?>
         <div id="delete-confirmation-wrapper" class="hidden">
             <div id="delete-confirmation">
-                <p>Are you sure you want to delete this appointment?</p>
+                <p>Are you sure you want to delete this grant?</p>
```

Comment:
Maybe add a comment stating what this does?

Reply:
Definitely! I will state that this is a confirmation forwarning the user that they are about to delete the grant.